admin_guide/manage_nodes.adoc

```diff
@@ -270,3 +270,26 @@ $ oadm manage-node <node1> <node2> \
 Alternatively, instead of specifying specific node names (e.g., `_<node1>_
 _<node2>_`), you can use the `--selector=_<node_selector>_` option to evacuate
 pods on selected nodes.
+
+== Configuring Node Resources
+Edit your node configuration file by passing kubelet arguments:
```

Comment:
Maybe: "You can configure node resources by adding kubelet arguments to the <appropriate section of the node config>."